packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -114,7 +114,7 @@ describe(`background sync queue`, function() {
       await backgroundSyncQueue.replayRequests();
       throw new Error('Replay should have failed because of invalid URL');
     } catch (err) {
-      expect(err[0].status).to.equal(404);
+      expect(err).to.be.an('Array');
```

Comment:
This change still worries me - a lot.

Why would err be an array? Shouldn't it just be a single error that is thrown?

You should be checking for an expected, specific error.

To unblock could you change this to:

```
return backgroundsyncQueue.replayRequests()
    .then(() => {
        throw new Error('Expected an error to be thrown);
    }, (err) => {
        // TODO: Check for specific error.
    });
```

Reply:
done